web/Web/web/static/js/document-viewer.js

```diff
@@ -476,6 +477,22 @@ docView.prototype = {
           Array.from(document.getElementById('scal-info').getElementsByTagName('small')).forEach((span, i) => {
             span.innerHTML = temp[i]
           })
+
+          if (options.logDSInfoURL !== null) {
+            if (result['stratum_size'] === parent.viewStack.length + 1) {
```

Comment:
There's a bug here. After 9 or 10 sratums the ds_logging urls stops getting called. I think its because viewstack is the number of samples not stratum size, but I'm not sure.